jdaviz/configs/default/plugins/plot_options/plot_options.py

```diff
@@ -661,6 +664,73 @@ def _update_stretch_histogram(self, msg={}):
         # update the n_bins since this may be a new layer
         self._histogram_nbins_changed()
 
+    @observe('is_active', 'image_color_mode_value', 'image_color_value', 'image_colormap_value',
+             'image_contrast_value', 'image_bias_value',
+             'stretch_function_value', 'stretch_vmin_value', 'stretch_vmax_value',
+             'stretch_hist_nbins', 'stretch_hist_zoom_limits')

```

Comment:
You tell me. Colorbar needs to track changes in the histogram X-axis too. So unless these nbins also trigger one of the other things it is already listening, I need it. Not sure what `stretch_hist_zoom_limits` is doing.

Reply:
Since `x` is currently sampled on the bins (see conversation below), we do need to keep `stretch_hist_nbins` so that the resolution of the colorbar is controlled by the same input.  But I don't see any need for `stretch_hist_zoom_limits` (which just changes which data is included in the histogram, but shouldn't affect the bins themselves (unless perhaps that would break if some bins on the extremes are unused - worth testing first!)

Reply:
Actually I think we still need it because the display histogram X-axis values change when you toggle follow zoom, so we also need to update the colorbar.